spec/models/event_spec.rb

```diff
@@ -121,4 +121,23 @@
       end
     end
   end
+
+  describe '#upcoming?' do
+    subject(:event_upcoming?) { event.upcoming? }
```

Comment:
If you're not going to use the named subject you can just do `subject { event.upcoming? }`.